test/bin.spec.js

```diff
@@ -1093,6 +1093,24 @@ export const testCan = {
 
     assert.ok(rm.status.success())
   }),
+  'can filecoin info with not found': test(async (assert, context) => {
```

Comment:
given this has abstractions from APIs, I can't simply put something in store to test success case. Leaving a failured piece not found for now and will improve tests later on